Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -262,6 +262,21 @@ public let CodeGenerators: [CodeGenerator] = [
         b.callMethod(methodName!, on: obj, withArgs: arguments)
     },
 
+    CodeGenerator("MethodCallWithSpreadGenerator", input: .object()) { b, obj in
+        var methodName = b.type(of: obj).randomMethod()
+        if methodName == nil {
+            guard b.mode != .conservative else { return }
+            methodName = b.genMethodName()
+        }
+        guard let arguments = b.randCallArguments(forMethod: methodName!, on: obj) else { return }
```

Comment:
I think we don't need `b.randCallArguments(forMethod:,on:)` here and in the other `XWithSpread` generators since we will (hopefully) be spreading some of the arguments, in which case the following arguments are anyway at the wrong position (instead, probably just pick 3-5 random variables). Due to that, I think we might anyway only want to run these `XWithSpread` CodeGenerators if we are not in conservative mode, since we are currently unable to track element types of arrays. So lets put the `guard b.mode != .conservative else { return }` at the start (and add a short comment in front of it). That will also simplify the first 5 lines of this function, and we can then also drop the `b.type(of: arg).Is(.iterable)` requirement I think (but then decrease the spread probability, maybe to 0.25?).

Reply:
done